mayastor/tests/error_count_retry.rs

```diff
@@ -1,139 +1,141 @@
-extern crate log;
-
 pub use common::error_bdev::{
     create_error_bdev,
     inject_error,
-    SPDK_BDEV_IO_TYPE_READ,
     SPDK_BDEV_IO_TYPE_WRITE,
     VBDEV_IO_FAILURE,
 };
 use mayastor::{
-    bdev::{nexus_create, nexus_lookup, ActionType, NexusErrStore, QueryType},
+    bdev::{nexus_create, nexus_lookup, NexusStatus},
     core::{Bdev, MayastorCliArgs},
     subsys::Config,
 };
 
-pub mod common;
-
-static ERROR_COUNT_TEST_NEXUS: &str = "error_count_retry_nexus";
+use common::MayastorTest;
+use once_cell::sync::OnceCell;
 
-static DISKNAME1: &str = "/tmp/disk1.img";
+pub mod common;
 
-static ERROR_DEVICE: &str = "error_retry_device";
-static EE_ERROR_DEVICE: &str = "EE_error_retry_device"; // The prefix is added by the vbdev_error module
-static BDEV_EE_ERROR_DEVICE: &str = "bdev:///EE_error_retry_device";
+static YAML_CONFIG_FILE: &str = "/tmp/error_retry_test.yaml";
+static MS: OnceCell<MayastorTest> = OnceCell::new();
 
-static YAML_CONFIG_FILE: &str = "/tmp/error_count_retry_nexus.yaml";
-#[ignore]
-#[tokio::test]
-async fn nexus_error_count_retry_test() {
-    common::truncate_file(DISKNAME1, 64 * 1024);
+static NON_ERROR_DISK: &str = "/tmp/non_error.img";
+static ERROR_DISK: &str = "/tmp/error.img";
+static NON_ERROR_BASE_BDEV: &str = "aio:////tmp/non_error.img?blk_size=512";
```

Comment:
Is there an extra slash '/' before tmp? I usually see 3 i.e. aio:///tmp/....

Reply:
Now fixed, thanks